tpgtools/overrides/firebaserules/samples/release/firestore_release.tf.tmpl

```diff
@@ -1,22 +1,22 @@
+# In case there are existing rules in the default db
+import {
+  id = "projects/{{project}}/releases/cloud.firestore"
```

Comment:
(Just to be clear, I'm not saying this _shouldn't_ be here, just want to make sure I understand why it's here & that it won't have unintended side effects on other tests)

Reply:
1. Yes, it relies on the default Firestore instance being present and having rules already.
2. It's unlikely to break other tests because no other tests depend on the rules.

I'm trying to figure out what the rules state is in the test project & why it has rules already.

Reply:
I discovered the Google Cloud Console UI automatically deploys default rules if none present, and there's no way to delete that other than via the API. How can I clean this up for all the test projects?

Reply:
Sorry for the delayed response! How would you feel about making the tests create new projects to run the tests in? That would avoid the problem of "someone visited the UI" because nobody will ever do that.

Reply:
https://github.com/GoogleCloudPlatform/magic-modules/blob/main/tpgtools/README.md?plain=1#L234 has more information about splitting doc / test examples

Reply:
It looks to me like `random_suffix` will be autogenerated for variables (similar to how it works for Magic Modules). For example, you can see in the [generated code for this PR](https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-12263-old..auto-pr-12263#diff-764e78149213c9818626c89ec8b987fa9b1d5e24daa20a8c3c84b080a449074eR136) that there's a random_suffix appended to the database name.

That said, if everything's covered by `firestore_release_additional`, and it doesn't have this problem, I'm happy for that to cover our testing needs and leave the other one docs-only.